ccmlib/scylla_repository.py

```diff
@@ -122,3 +126,10 @@ def __get_dir():
     if not os.path.exists(repo):
         os.mkdir(repo)
     return repo
+
+
+def run_scylla_install_script(install_dir, target_dir):
+    subprocess.check_call(['bash', '-c', ''' sed 's|"$prefix|"$root/$prefix|' -i install.sh '''], cwd=install_dir)
+    subprocess.check_call(['bash', '-c', ''' {0}/install.sh --root {1}  --target  centos --disttype redhat --pkg server '''.format(install_dir, target_dir)], cwd=install_dir)
+    subprocess.check_call(['bash', '-c', 'mkdir -p {0}/conf; cp ./conf/scylla.yaml {0}/conf'.format(target_dir)], cwd=install_dir)
+    subprocess.check_call(['bash', '-c', 'ln -s {}/opt/scylladb/bin bin'.format(target_dir)], cwd=target_dir)
```

Comment:
This might silently break in the future in case you already have a `./bin` directory.
How about `ln -s {}/opt/scylladb/bin .`?